### PR TITLE
Change "social web" to "fediverse" in a few banners in web UI

### PR DESCRIPTION
**Pull request by Gargron**
And increase the brightness on the sign in banner in the sidebar.

### CODE REVIEW

File: app/javascript/mastodon/features/explore/links.jsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ class Links extends PureComponent {
 
     const banner = (
       <DismissableBanner id='explore/links'>
-        <FormattedMessage id='dismissable_banner.explore_links' defaultMessage='These are news stories being shared the most on the social web today. Newer news stories posted by more different people are ranked higher.' />
+        <FormattedMessage id='dismissable_banner.explore_links' defaultMessage='These news stories being shared the most on the fediverse today. Newer news stories posted by more different people are ranked higher.' />
```
Comment (ClearlyClaire) [inline app/javascript/mastodon/features/explore/links.jsx:48]: The removal of “are” kinds of break the sentence here?
> These news stories being shared the most on the fediverse today.

File: app/javascript/styles/mastodon/components.scss [hunk 2]
```diff
@@ -968,7 +968,7 @@ body > [data-popper-placement] {
   p {
     font-size: 15px;
     line-height: 22px;
-    color: $darker-text-color;
+    color: $secondary-text-color;
```
Comment (ClearlyClaire) [inline app/javascript/styles/mastodon/components.scss:971]: Unrelated change?

File: app/javascript/mastodon/features/explore/links.jsx (hunk 1 above)
Comment (Gargron) [reply app/javascript/mastodon/features/explore/links.jsx]: This was an oversight in the `defaultMessage` (but not the actual message in `en.json`). However, fixed.

File: app/javascript/styles/mastodon/components.scss (hunk 2 above)
Comment (Gargron) [reply app/javascript/styles/mastodon/components.scss:971]: Intended to increase readability.

**Review by ClearlyClaire — APPROVED** [review]
